types/kv-model.d.ts

```diff
@@ -49,13 +49,43 @@ export declare class KeyValueModel extends ModelBase {
    *
    * @header KeyValueModel.set(key, value, cb)
    */
-  set(
+  static set(
     key: string,
     value: KVData,
     options?: Options,
     callback?: Callback<boolean>,
   ): PromiseOrVoid<boolean>;
 
+  /**
+   * Delete the key-value pair associated to the given key.
+   *
+   * @param {String} key Key to use when searching the database.
+   * @options {Object} options
+   * @callback {Function} callback
+   * @param {Error} err Error object.
+   * @param {*} result Value associated with the given key.
+   * @promise
+   */
+  static delete(
+    key: string,
+    options?: Options,
+    callback?: Callback<void>,
+  ): PromiseOrVoid<void>;
+
+  /**
+   * Delete all keys (and values) associated to the current model.
+   *
+   * @options {Object} options Unused ATM, placeholder for future options.
+   * @callback {Function} callback
+   * @param {Error} err Error object.
+   * @promise
+   */
+  static deleteAll(
+    key: string,
```

Comment:
Don't think it's supposed to take a `key` argument for `deleteAll`.

Reply:
Good catch. Will fix.